peachjam/settings.py

```diff
@@ -432,3 +432,6 @@
             "LOCATION": "/var/tmp/django_cache",
         },
     }
+
+# Override X-Frame-Options header value
+X_FRAME_OPTIONS = "ALLOW"
```

Comment:
This will allow any site to place our content in frames, which is a bit dangerous. I think `SAMEORIGIN` is a better idea -- only our pages can embed our content.  https://docs.djangoproject.com/en/3.2/ref/clickjacking/